### PR TITLE
[CUBRIDMAN-95] 11.2 release note

### PR DESCRIPTION
**Pull request by ssihil**
http://jira.cubrid.org/browse/CUBRIDMAN-95

### CODE REVIEW

File: en/release_note/release_note_latest_ver.rst [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+11.2 Release Notes
 ******************
 
 .. contents::
 
 Release Notes Information
 =========================
 
-This document includes information on CUBRID 11.0(Build Number: 11.0.0.0248-b53ae4a).
+This document includes information on CUBRID 11.2(Build Number: 11.2.0.0645-8cdfd54).
```
Comment (mhoh3963) [inline en/release_note/release_note_latest_ver.rst]: If you describe the build number, I think the manual is dependent on the build number.
So how about removing the build number?

File: en/release_note/release_note_latest_ver.rst [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 
-CUBRID 11.0 includes all of the fixed errors and improved features that were detected in the CUBRID 10.2 and were applied to the previous versions.
+CUBRID 11.2 includes all of the fixed errors and improved features that were detected in the CUBRID 11.0 and were applied to the previous versions.
+
+For CUBRID 11.0, please find https://www.cubrid.org/manual/en/11.0/release_note/index.html.
```
Comment (charischau) [inline en/release_note/release_note_latest_ver.rst:16]:
```suggestion
For information of CUBRID 11.0, please find at https://www.cubrid.org/manual/en/11.0/release_note/index.html.
```

File: en/release_note/release_note_latest_ver.rst [hunk 3]
```diff
@@ -24,70 +26,70 @@ For CUBRID 9.3, please find https://www.cubrid.org/manual/en/9.3.0/release_note/
...
 * is more stable, faster, and more convenient for administrators.
 * fixes a large number of critical bugs.
-* includes useful SQL extensions: Supporting RVC (Row Value Constructor) and various REGEXP functions.
+* includes useful SQL extensions: User Schema concept and Synonym are supported.
 * includes code refactoring and modernization.
 
-CUBRID 11.0 **improves security** by providing data encryption and packet encryption. This version prevents abnormal data loss by supporting table-based TDE (Transparent Data Encryption) and packet encryption between the driver and server.
+CUBRID 11.2 **improved connectivity** by providing database link between homogeneous and heterogeneous DBMS. In addition, this version provides CDC (Change data capture) function by supporting supplemental logging that can track changed data.
+
+CUBRID 11.2 is **faster**. This version improves query optimization such as predicate pushdown and view transformation, supports Hash List Scan for all subqueries, improves performance of count(*) without conditional clause using cache and improves truncate table to further improve performance.
```
Comment (charischau) [inline en/release_note/release_note_latest_ver.rst]:
```suggestion
CUBRID 11.2 is **faster**. This version improves query optimization such as predicate pushdown and view transformation, supports Hash List Scan for all subqueries, improves performance of count(*) without conditional clause using cache, and improves truncate table to further improve performance.
```

**Review by charischau — APPROVED** [review]

File: en/release_note/release_note_latest_ver.rst (hunk 1 above)
Comment (ssihil) [reply en/release_note/release_note_latest_ver.rst]: removed as suggested.

File: en/release_note/release_note_latest_ver.rst (hunk 2 above)
Comment (mhoh3963) [inline en/release_note/release_note_latest_ver.rst:16]: 11.0 -> 11.2 ?

Comment (ssihil) [reply en/release_note/release_note_latest_ver.rst:16]: This is the format that has been used since the previous version, so I will leave it as it is.

Comment (charischau) [reply en/release_note/release_note_latest_ver.rst:16]: Okay!

File: en/release_note/release_note_latest_ver.rst [hunk 4]
```diff
@@ -24,70 +26,70 @@ For CUBRID 9.3, please find https://www.cubrid.org/manual/en/9.3.0/release_note/
...
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Modified to recognize the space character at the end of the string, it is recognized as a different character string according to the space character at the end of the string.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Due to the change in the statistics collection method, it is necessary to perform periodic statistics collection.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Existing Cautions
 -----------------
 
+When creating a table without an option, it is created as a reuse_oid table
```
Comment (mhoh3963) [inline en/release_note/release_note_latest_ver.rst:121]: You need to add something what change at v11.2.
ex) directory of cci driver ..

File: en/release_note/release_note_latest_ver.rst (hunk 3 above)
Comment (ssihil) [reply en/release_note/release_note_latest_ver.rst]: reflected. Thanks for the suggestion.

File: ko/release_note/release_note_latest_ver.rst [hunk 5]
```diff
@@ -25,70 +27,108 @@ CUBRID 9.3에 대한 정보는 https://www.cubrid.org/manual/ko/9.3.0/release_no
...
+.. _11_2_new_cautions:
 
 신규 주의 사항
 --------------
 
-CUBRID 11.0의 데이터베이스 볼륨은 CUBRID 10.2 및 그 이전 버전의 볼륨과 호환되지 않는다. 
+CUBRID 11.2의 데이터베이스 볼륨은 CUBRID 11.1 및 그 이전 버전의 볼륨과 호환되지 않는다. 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-옵션 없이 테이블 생성 시 reuse_oid 테이블로 생성된다.
+User schema 개념 도입으로 사용자별로 동일한 객체명을 사용할 수 있도록 변경됨
```
Comment (mhoh3963) [inline ko/release_note/release_note_latest_ver.rst]:
```suggestion
User schema 개념 도입으로 사용자 별로 동일한 객체명을 사용할 수 있으며 지원에 따라 다음 동작이 변경됨.
```

File: ko/release_note/release_note_latest_ver.rst [hunk 6]
```diff
@@ -25,70 +27,108 @@ CUBRID 9.3에 대한 정보는 https://www.cubrid.org/manual/ko/9.3.0/release_no
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* 객체명에 "."(dot)을 허용하지 않는다.
+* 질의 또는 유틸리티 명령어 사용 시 "[사용자명].객체명"으로 사용해야 한다. (단, 로그인 된 사용자의 객체를 질의하는 경우에는 사용자명을 생략할 수 있음)
+* info schema, show full tables 결과에 사용자명 포함되도록 변경되었다.
+* 11.2 이전 loaddb 파일은 11.2에서 수행할 수 있도록 user명.table명으로 수정하거나 -no-user-specified-name  옵션을 설정하여 loaddb를 수행할 수있다.
```
Comment (mhoh3963) [inline ko/release_note/release_note_latest_ver.rst]:
```suggestion
* 11.2 이전 loaddb 파일은 11.2에서 수행할 수 있도록 user명.table명으로 수정하거나 -no-user-specified-name  옵션을 설정하여 loaddb를 수행할 수 있다.
```

File: ko/release_note/release_note_latest_ver.rst [hunk 7]
```diff
@@ -25,70 +27,108 @@ CUBRID 9.3에 대한 정보는 https://www.cubrid.org/manual/ko/9.3.0/release_no
...
+
+JavaSP의 defaultconnection 사용 시 다음 함수 및 동작 변경됨
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* java.sql.DatabaseMetaData의 모든 function 지원하지 않는다.
+* java.sql.Connection의 createClob(), createBlob() 지원하지 않는다.
+* java.sql.Statement의 addBatch(), clearBatch(), executeBatch(), setMaxRows(), cancel() 지원하지 않는다.
+* 하나의 prepare에 여러개의 질의 수행을 지원하지 않는다.
```
Comment (mhoh3963) [inline ko/release_note/release_note_latest_ver.rst]:
```suggestion
* 하나의 prepare (또는 execute)에 multiple sql을 지원하지 않는다.
```

File: ko/release_note/release_note_latest_ver.rst [hunk 8]
```diff
@@ -25,70 +27,108 @@ CUBRID 9.3에 대한 정보는 https://www.cubrid.org/manual/ko/9.3.0/release_no
...
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-문자열 끝의 공백 문자를 인식하게 수정하여, 문자열 끝의 공백 문자에 따라 다른 문자열로 인식된다.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-통계 수집 방식의 변경으로 주기적인 통계 수집 수행이 필요하다.
+
+* UPDATE t1 SET c1 = 9 WHERE c1; 형태로 사용하면 오류 발생한다.
+
+Multiple SQL는 반드시 세미콜론으로 구분하도록 변경됨
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+CCI Driver 디렉토리가 $CUBRID/lib, $CUBRID/include에서 $CUBRID/cci/lib, $CUBRID/cci/include로 각각 변경됨
```
Comment (mhoh3963) [inline ko/release_note/release_note_latest_ver.rst]:
```suggestion
CUBRID 패키지내의 CCI Driver 디렉토리가 $CUBRID/lib, $CUBRID/include에서 $CUBRID/cci/lib, $CUBRID/cci/include로 각각 변경됨
```

File: ko/release_note/release_note_latest_ver.rst [hunk 9]
```diff
@@ -25,70 +27,108 @@ CUBRID 9.3에 대한 정보는 https://www.cubrid.org/manual/ko/9.3.0/release_no
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* LD_LIBRARY_PATH에 $CUBRID/cci/lib를 추가해야 한다.
```
Comment (mhoh3963) [inline ko/release_note/release_note_latest_ver.rst]:
```suggestion
* CCI 사용시 환경변수에 LD_LIBRARY_PATH에 $CUBRID/cci/lib를 추가해야 한다.
```

File: ko/release_note/release_note_latest_ver.rst (hunk 5 above)
Comment (ssihil) [reply ko/release_note/release_note_latest_ver.rst]: How about changing to "User schema 개념 도입으로 사용자 별로 동일한 객체명을 사용할 수 있으며, 지원에 따라 다음 동작이 변경됨" ?

Comment (mhoh3963) [reply ko/release_note/release_note_latest_ver.rst]: ok

File: ko/release_note/release_note_latest_ver.rst (hunk 6 above)
Comment (ssihil) [reply ko/release_note/release_note_latest_ver.rst]: reflected.

File: ko/release_note/release_note_latest_ver.rst (hunk 7 above)
Comment (ssihil) [reply ko/release_note/release_note_latest_ver.rst]: reflected.

File: ko/release_note/release_note_latest_ver.rst (hunk 8 above)
Comment (ssihil) [reply ko/release_note/release_note_latest_ver.rst]: reflected.

File: ko/release_note/release_note_latest_ver.rst (hunk 9 above)
Comment (ssihil) [reply ko/release_note/release_note_latest_ver.rst]: reflected.

**Review by kisoo-han — APPROVED** [review]

File: en/release_note/release_note_latest_ver.rst [hunk 10]
```diff
@@ -24,70 +26,108 @@ For CUBRID 9.3, please find https://www.cubrid.org/manual/en/9.3.0/release_note/
...
 * is more stable, faster, and more convenient for administrators.
 * fixes a large number of critical bugs.
-* includes useful SQL extensions: Supporting RVC (Row Value Constructor) and various REGEXP functions.
+* includes useful SQL extensions: User Schema concept and Synonym are supported.
 * includes code refactoring and modernization.
 
-CUBRID 11.0 **improves security** by providing data encryption and packet encryption. This version prevents abnormal data loss by supporting table-based TDE (Transparent Data Encryption) and packet encryption between the driver and server.
+CUBRID 11.2 **improved connectivity** by providing database link between homogeneous and heterogeneous DBMS. In addition, this version provides CDC (Change data capture) function by supporting supplemental logging that can track changed data.
+
+CUBRID 11.2 is **faster**. This version improves query optimization such as predicate pushdown and view transformation, supports Hash List Scan for all subqueries, improves performance of count(*) without conditional clause using cache, and improves truncate table to further improve performance.
```
Comment (hornetmj) [inline en/release_note/release_note_latest_ver.rst]: From 'without' to the end of the sentence, it is shown in italic style. Is it intended?

File: ko/release_note/release_note_latest_ver.rst [hunk 11]
```diff
@@ -25,70 +27,108 @@ CUBRID 9.3에 대한 정보는 https://www.cubrid.org/manual/ko/9.3.0/release_no
...
 * 보다 안정적이고, 빠르고, 관리자의 편의성을 높였다. 
 * 수많은 중요 버그가 수정되었다.
-* 유용한 SQL 확장: RVC(Row Value Constructor) 및 다양한 REGEXP 함수를 지원한다.
+* 유용한 SQL 확장: User Schema 개념, Synonym을 지원한다.
 * 코드 재작성 및 최신화 작업을 포함한다.
 
-CUBRID 11.0은 데이터 암호화와 패킷 암호화를 제공하여 **보안을 향상시켰다**. 이 버전은 테이블 기반의 TDE(Transparent Data Encryption) 지원과 드라이버와 서버 사이의 패킷 암호화를 지원하여 비정상적으로 데이터가 유출되는 것을 방지하였다.
+CUBRID 11.2는 동종 DBMS와 이종 DBMS간의 데이터베이스 링크를 제공하여 **연계성을 향상시켰다**. 또한 이 버전에서는 변경된 데이터를 추적할 수 있는 supplemental logging을 지원하여 CDC(Change data capture) 기능을 제공한다.
+
+CUBRID 11.2은 **더 빨라졌다**. 이 버전은 조건절 푸시다운(predicate pushdown) 과 뷰 변환(view transformation) 등의  질의 최적화(query optimization) 개선, 모든 서브질의에 Hash List Scan 지원,  캐시를 이용한 조건절 없는 count(*) 성능 개선 및 truncate table을 개선하여 성능을 더욱 향상시켰다.
```
Comment (hornetmj) [inline ko/release_note/release_note_latest_ver.rst]: ') 성능 개선 및 truncate table을 개선하여 성능을 더욱 향상시켰다.' looks as italic style.

**Review by hornetmj — APPROVED** [review]

**Review by beyondykk9 — APPROVED** [review]

File: en/release_note/release_note_latest_ver.rst (hunk 10 above)
Comment (ssihil) [reply en/release_note/release_note_latest_ver.rst:43]: The reason is unknown, but it is shown as italic style only in the code view. In 'view file', it is displayed normally.

File: ko/release_note/release_note_latest_ver.rst (hunk 11 above)
Comment (ssihil) [reply ko/release_note/release_note_latest_ver.rst:44]: The reason is unknown, but it is shown as italic style only in the code view. In 'view file', it is displayed normally.

File: en/release_note/release_note_latest_ver.rst (hunk 10 above)
Comment (ssihil) [reply en/release_note/release_note_latest_ver.rst]: reflected.

File: ko/release_note/release_note_latest_ver.rst (hunk 11 above)
Comment (ssihil) [reply ko/release_note/release_note_latest_ver.rst]: reflected.

**Pull request merged**